ta/Makefile

```diff
@@ -4,25 +4,4 @@ CPPFLAGS += -DCFG_TEE_TA_LOG_LEVEL=$(CFG_TEE_TA_LOG_LEVEL)
 # The UUID for the Trusted Application
 BINARY=8aaaf200-2450-11e4-abe2-0002a5d5c51b
 
--include $(TA_DEV_KIT_DIR)/mk/ta_dev_kit.mk
```

Comment:
From https://github.com/OP-TEE/optee_os/pull/1276 (when a target starts with `./`, `$@` does not), it seems like above is expected behavior?

When `O` is not specified (or specified as `.`), `out-dir` is `.`, and in `<ta_dev_kit_dir>/mk/link.mk`, `link-out-dir = $(out-dir)` and the `$(link-out-dir)/$(binary).*` targets just become `./$(binary).*`, so `$(BINARY).ta` = `$(binary).ta` = `./$(binary).ta` is a valid target?

But when `O` is specified as something other than `.`, the there's no such target in `link.mk`.